src/pages/blog/starlingx-deploying-your-app-3.md

```diff
@@ -0,0 +1,250 @@
+---
+templateKey: blog-post
+title: Deploying your application as a StarlingX Application
+author: Tomás N. P. Barros
+date: 
+category:
+  - label: Features & Updates
+---
+
+Deploying your application as a StarlingX application is the highest level of integration you can achieve with the platform. It is a step up from deploying an app via [Helm](https://www.starlingx.io/blog/starlingx-deploying-your-app-1/) or [FluxCD](https://www.starlingx.io/blog/starlingx-deploying-your-app-2/). In this post, I will guide you through the process of deploying the [demo app](https://github.com/bmuniz-daitan/poc-starlingx-messages) used in previous deployment posts, along with some basic management commands <!- more ->.
+
+It is worth noting that while deploying a containerized app as a StarlingX app has its advantages and is easy to do, transforming a containerized application into a StarlingX app might not be. Therefore, Helm and FluxCD options are more beginner-friendly and are the typical choice for developers and users who don't plan to create an application that will be officially bundled into the StarlingX project.
+ 
+## Prerequisites
+
+This tutorial expects that your application will be containerized and with ready helm-charts. If this is not your case, please read the blog post already mentioned about Helm.
```

Comment:
The thing is, the previous blog posts also expects the application to be containerized, the posts only guide the deployment and not the process of creating the Helm charts